datastore/tests/unit/test_transaction.py

```diff
@@ -15,7 +15,30 @@
 import unittest
 
 import mock
+from google.cloud.datastore.transaction import TransactionOptions
 
+class TestTransactionOptions(unittest.TestCase):
```

Comment:
There should be two empty lines between imports and your defined members (like classes and functions). I recommend reading:

- https://google.github.io/styleguide/pyguide.html
- https://www.python.org/dev/peps/pep-0008/

Reply:
ok, I thought nox lint would catch it.

Reply:
Good point, not sure why it didn't. Still worth knowing.